src/commands/tileindex-validate/__test__/tileindex.validate.test.ts

```diff
@@ -36,13 +36,17 @@ describe('getTileName', () => {
     assert.equal(convertTileName('CH11_1000_0501', 5000), 'CH11_5000_0101');
     assert.equal(convertTileName('CH11_1000_0505', 5000), 'CH11_5000_0101');
   });
-
   it('should get correct parent tile 1:10k', () => {
     assert.equal(convertTileName('CH11_1000_0101', 10000), 'CH11_10000_0101');
     assert.equal(convertTileName('CH11_1000_0110', 10000), 'CH11_10000_0101');
     assert.equal(convertTileName('CH11_1000_1010', 10000), 'CH11_10000_0101');
     assert.equal(convertTileName('CH11_1000_1001', 10000), 'CH11_10000_0101');
   });
+  it('should get correct parent tile 1:50k', () => {
```

Comment:
1:50k tile should be just the mapsheet name. `CK08.tiff`

Reply:
It does generate the correct name from the map sheet further along in the process.

Reply:
I dont think we should be generating incorrect names anywhere, this could really bite us in the future if we use this function for naming...